### PR TITLE
fix: sign typed data 

### PR DESCRIPTION
**Pull request by bluecco**

### CODE REVIEW

File: src/__tests__/fixture.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+export const executionFromOusideTypedData = (to: string) => ({
```
Comment (hwildwood) [inline src/__tests__/fixture.ts:1]: $nitpic

Technically this is the typed data for the OutsideExecution struct so `outsideExecutionTypedData` would be more accurate.

https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-9.md#1-build-the-outsideexecution-struct

Comment (bluecco) [reply src/__tests__/fixture.ts:1]: ah yep, will refactor

File: src/sessionDappService.ts [hunk 2]
```diff
@@ -359,29 +398,28 @@ export class SessionDappService {
     }
   }
 
-  private async compileSessionSignatureFromOutside(
+  private async compileSessionTxSignatureFromOutside(
```
Comment (hwildwood) [inline src/sessionDappService.ts:401]: $suggestion

`compileSessionSignatureFromOutsideExecution`

**Review by hwildwood — APPROVED** [review]
Nice! LGTM ✅

**Review by vladutjs — APPROVED** [review]

**Pull request merged**